integration/testnet/network.go

```diff
@@ -866,7 +871,7 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 			net.AccessPortsByContainerName[nodeContainer.Name()] = hostGRPCPort
 			net.AccessPorts[AccessNodeAPIProxyPort] = hostHTTPProxyPort
 
-			if nodeConf.SupportsUnstakedNodes {
+			if nodeConf.SupportsObserverNodes || nodeConf.Role == flow.RoleObserverService {
```

Comment:
in the observer case, you'll need to set the `staked` to false, and the `bootstrap-node-*` params to the address and key of one of the access nodes

Reply:
checkout the description of this PR for the list of require params for running an observer:
https://github.com/onflow/flow-go/pull/2037

Reply:
Thank you for the review. I need some additional testing on this one. It may take a couple of calendar days. We need to sync internally what is right ordering and merger with the next sprint.